wordpress currently does not work with 0.9

gitricko opened the pull request:
Below changes will work. 

Also minus 1 command

gitricko committed:
change default

gitricko committed:
use env

gitricko committed:
use sprintf

gitricko committed:
Update wp-config.php

gitricko committed:
Update README.md

gitricko commented:
@juadk

juadk reviewed (approved):
Tested successfully with latest Epinio, thanks.

Could you also update the title please? https://github.com/epinio/example-wordpress/blob/main/README.md#step-6---create-application-bind-the-database-service-and-update-configuration 
By removing `and update the configuration`
Otherwise, I will do it once the PR merged, no worries.

gitricko committed:
Update README.md

Could you also update the title please? https://github.com/epinio/example-wordpress/blob/main/README.md#step-6---create-application-bind-the-database-service-and-update-configuration
By removing and update the configuration
Otherwise, I will do it once the PR merged, no worries.

gitricko commented:
> Tested successfully with latest Epinio, thanks.
> 
> Could you also update the title please? https://github.com/epinio/example-wordpress/blob/main/README.md#step-6---create-application-bind-the-database-service-and-update-configuration By removing `and update the configuration` Otherwise, I will do it once the PR merged, no worries.

done... please merge
